src/main/java/uk/ac/sanger/sccp/stan/model/DnapStudy.java

```diff
@@ -12,23 +12,21 @@ public class DnapStudy implements HasIntId, HasEnabled, HasName {
     @Id
     @GeneratedValue(strategy= GenerationType.IDENTITY)
     private Integer id;
+    private Integer ssId;
     private String name;
     private boolean enabled = true;
 
     public DnapStudy() {}
 
-    public DnapStudy(Integer id, String name, boolean enabled) {
+    public DnapStudy(Integer id, Integer ssId, String name, boolean enabled) {
         this.id = id;
```

Comment:
Do we need this `id` field now with the new `ssId` field added?

Reply:
It's kept to make the transition easier. The existing works linked to dnap studies refer to them by this id, since it's the primary key of the table.

Reply:
Ahh, ok. that make sense 👍